src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  @for $i from 1 through 5 {
+    &--#{$i} &__star:nth-child(-n + #{$i}) {
+      background-image: url('../images/star-active.svg');
+    }
+  }
+
+  &__star {
+    background-image: url('../images/star.svg');
+    background-position: center;
+    background-repeat: no-repeat;
+    width: $stars-size;
+    height: $stars-size;
+
+    &:not(:last-child) {
+      margin-right: 4px;
```

Comment:
The `margin-right` value should be extracted to a variable in `variables.scss` to promote reusability and maintainability, and to adhere to the DRY principle.